client/state/preferences/schema.js

```diff
@@ -73,7 +73,15 @@ export const remoteValuesSchema = {
 		},
 		colorScheme: {
 			type: 'string',
-			enum: [ 'default', 'light', 'dark', 'classic-blue', 'classic-bright', 'laser-black' ],
+			enum: [
+				'default',
+				'light',
+				'dark',
```

Comment:
Some mapping of the legacy values (`default`, `light`, `dark`) to the new ones would be nice to have. Not now, but in near future.